### PR TITLE
Update proofValue test title & assertions.

### PR DESCRIPTION
**Pull request by aljones15**
wording of this statement might be changed shortly.

### CODE REVIEW

**Review by PatStLouis — CHANGES_REQUESTED** [review]
Most of my comments are around these tests expecting proofs will always have a cryptosuite

File: suites/create.js [hunk 1]
```diff
@@ -194,34 +195,33 @@ export function runDataIntegrityProofFormatTests({
...
-            expectedPrefix,
-            `Expected "proof.proofValue" to be a ${encodingName} value`
-          );
-
-        isValidMultibaseEncoded(proof.proofValue, expectedPrefix).should
-          .equal(
-            true,
-            `Expected "proof.proofValue" to be a valid ${encodingName} value`
-          );
+        shouldHaveProofValue({proof, expectedPrefix, encodingName});
```
[the remark this answers is not in the document]
Comment (BigBlueHat) [reply suites/create.js:224]: @PatStLouis why remove this one?

File: suites/create.js [hunk 2]
```diff
@@ -194,34 +195,33 @@ export function runDataIntegrityProofFormatTests({
...
-          // assert that it is a string here.
-          proof.proofValue.should.be.a('string');
+          should.exist(proof, 'Expected proof to exist.');
+          should.exist(proof.cryptosuite,
+            'Expected proof to have property "cryptosuite".');
+          const {
+            prefix: expectedPrefix,
+            name: encodingName
+          } = expectedMultibasePrefix(proof.cryptosuite);
+          shouldHaveProofValue({proof, expectedPrefix, encodingName});
```
[the remark this answers is not in the document]
Comment (aljones15) [reply suites/create.js:209]: this actually already works with `Ed25519` https://github.com/w3c-ccg/data-integrity-test-suite-assertion/blob/3c53b6cf1e4db963c9ca5aa65b04e86000e65d8e/assertions.js#L41-L55

ed25519 is abs58 encoded.

https://www.w3.org/TR/vc-di-eddsa/#ed25519signature2020-0

File: suites/create.js [hunk 3]
```diff
@@ -194,34 +195,33 @@ export function runDataIntegrityProofFormatTests({
...
+    'Multibase of the Controller Documents 1.0 specification.', function() {
+      this.test.link = 'https://w3c.github.io/vc-data-integrity/#proofs:~:text=string%20value%20that%20contains%20the%20base%2Dencoded%20binary%20data%20necessary%20to%20verify%20the%20digital%20proof';
       for(const proof of proofs) {
+        should.exist(proof, 'Expected proof to exist.');
+        should.exist(proof.cryptosuite,
+          'Expected proof to have property "cryptosuite".');
         const {
           prefix: expectedPrefix,
           name: encodingName
         } = expectedMultibasePrefix(proof.cryptosuite);
```
[the remark this answers is not in the document]
Comment (aljones15) [reply suites/create.js:223]: this works with DataIntegrity or Ed25519Signature2020.

File: suites/create.js [hunk 4]
```diff
@@ -194,34 +195,33 @@ export function runDataIntegrityProofFormatTests({
...
+    it('("proof.proofValue") A string value that contains the base-encoded ' +
+    'binary data necessary to verify the digital proof using the ' +
+    'verificationMethod specified. The contents of the value MUST be ' +
+    'expressed with a header and encoding as described in Section 2.4 ' +
+    'Multibase of the Controller Documents 1.0 specification.', function() {
+      this.test.link = 'https://w3c.github.io/vc-data-integrity/#proofs:~:text=string%20value%20that%20contains%20the%20base%2Dencoded%20binary%20data%20necessary%20to%20verify%20the%20digital%20proof';
       for(const proof of proofs) {
+        should.exist(proof, 'Expected proof to exist.');
+        should.exist(proof.cryptosuite,
+          'Expected proof to have property "cryptosuite".');
```
[the remark this answers is not in the document]
Comment (aljones15) [reply suites/create.js]: this test is about proofValue so I just removed the cryptosuite check entirely: https://github.com/w3c-ccg/data-integrity-test-suite-assertion/pull/85/commits/a681268a0e7a6bab4621e2644fe8f5cab477b06a

File: suites/create.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (PatStLouis) [reply suites/create.js:209]: The issue is that `Ed25519Signature2020` doesn't use the `cryptosuite` field. [This section of the spec](https://www.w3.org/TR/vc-data-integrity/#dataintegrityproof), which the statement is from, is about `DataIntegrityProof` type, which `Ed25519Signature2020` isn't (it's `Ed25519Signature2020` type). I would expect the statements in the `DataIntegrityProof` suite section to only apply to proofs of type `DataIntegrityProof`.

File: suites/create.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (PatStLouis) [reply suites/create.js:224]: It's not been removed, it's been moved, see line 225 in the edit right above this one

File: suites/create.js (hunk 2 above)
Comment (aljones15) [reply suites/create.js]: addressed here: https://github.com/w3c-ccg/data-integrity-test-suite-assertion/pull/85/commits/d107aaf9958f324a04870972844da198d92ebc22

this is actually a problem in general for instance with the SD suites, some assertions are base proof only and some are derived proof only.

**Review by PatStLouis — APPROVED** [review]
I'm happy with these changes!

**Pull request merged**